sdk/src/beta9/terminal.py

```diff
@@ -28,7 +36,19 @@ def header(text: str, subtext: str = "") -> None:
     _console.print(header_text, subtext)
 
 
+def _add_footer_to_table(table: Table):
+    row_count = table.row_count
+    if not row_count:
+        table.add_row(*("-" * len(table.columns)))
+
+    table.add_section()
+    table.add_row(f"[bold]{row_count} items")
+
+
 def print(*objects: Any, **kwargs: Any) -> None:
+    for obj in objects:
+        if isinstance(obj, Table):
+            _add_footer_to_table(obj)
```

Comment:
This either needs to be removed or we need to remove all of the duplicate code that adds a count of items at the end of the table.